src/components/scenes/TransactionListScene.tsx

```diff
@@ -1,6 +1,8 @@
 import { FlashList } from '@shopify/flash-list'
 import { lt } from 'biggystring'
+import { asArray } from 'cleaners'
 import { EdgeCurrencyWallet, EdgeTokenMap, EdgeTransaction } from 'edge-core-js'
+import { asAssetStatus, AssetStatus } from 'edge-info-server/types'
```

Comment:
Crud. we need to add an optional `appId` field so we can optionally NOT hit all whitelabel apps with the same card.

Reply:
This will require a change to the cleaner in the info server.